test/test_autograd.py

```diff
@@ -2189,6 +2189,14 @@ def test_cat_empty(self):
                               lambda a, b: torch.cat((a, b)),
                               True, f_args_variable, f_args_tensor)
 
+    def test_trapz(self):
+        f_args_variable = (torch.randn(2, 3, requires_grad=True),
```

Comment:
Does adding a line to common_methods_invocations not work?

Reply:
Don't those have to be methods or is that function misnamed? `trapz` is only a function, not a method.

Reply:
Yeah,  you're right. `run_functional_checks` looks good then.